enclave/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "enclave"
+version = "0.1.0"
+authors = ["Michal Rostecki <mrostecki@opensuse.org>"]
+edition = "2018"
+
+[dependencies]
+anyhow = "1.0"
+byteorder = "1.4"
+chrono = "0.4"
+ctrlc = "3.1"
+libbpf-rs = "0.10"
+regex = "1.5"
+thiserror = "1.0"
```

Comment:
Have we checked for removing default features we are not using? Not a big deal if not. Might be an opportunity to reduce binary size.

Reply:
I didn't yet. I can do that.

Reply:
Is there any tool which automatically checks for possible default features removal?

Reply:
anyway, I'm done with manual feature removal ;)